management/telemetry/telemetry-dashboards/telemetry-dashboards.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Telemetry Dashboards
+permalink: /management/telemetry/telemetry-dashboards/
+---
+
+* TOC
+{:toc}
+
+***Note:** Initially, the Keboola platform was referred to as Keboola Connection (KBC). While it is now simply known as Keboola, references to “Connection” or the abbreviation “KBC” might still appear in table
+names, column names, etc.*
+
+To get to all three telemetry dashboards, click the top right corner of the profile picture. The [Project Consumption](#project-consumption) dashboard is always available. Check with your account manager to see 
+if [Organization Usage](#organization-usage) or [Activity Center](#activity-center) are also available. 
+
+## Project Consumption
+{: .image-popup}
+![Screenshot - Project Consumption Dashboard](/management/telemetry/telemetry-dashboards/project-consumption.png)
+
+By default, the dashboard is filtered for the last six months. The date range at the top of the dashboard can be changed. 
+
+The *KPIs* section presents overall statistics of the Keboola project. Some are self-explanatory (e.g., *Consumed Credits* and *Number of Jobs*). An explanation for the others can be found under 
+the question mark or here in the documentation. *Error Jobs Ratio* shows the number of error jobs as a percentage of the total number of jobs run. *Active Flows* shows the number of flows with jobs starting 
+in the selected date range. *Active Component Configurations* is like *Active Flows* but shows how many configurations have a job started in the selected date range, and the same applies to *Active 
+Transformations*.
+
+*Credits History* shows the evolution of consumed credits over time. *Trends* at the bottom of the print screen shows the monthly evolution of consumption broken into component types (bar chart) plus 
+a graph of the number of error jobs and the total number of jobs. 
+
+## Organization Usage
+The dashboard is filtered for the last six months, which can be adjusted. It shows data for the whole Keboola organization. If there are several projects for an account, it will show data for all of them. 
```

Comment:
Only first two sentences are correct. Then it should be:
If customer has an organization then the organization usage can be visible which usually means also more projects. Something like this. Hope it makes sense...